src/vmm/src/cpu_config/aarch64/custom_cpu_template.rs

```diff
@@ -217,7 +217,7 @@ mod tests {
                         {
                             "addr": "j",
                             "bitmap": "0bx00100xxx1xxxx00xxx1xxxxxxxxxxx1"
-                        },
+                        }
```

Comment:
hm, how did it use to work then?

Reply:
The parsing was failing before it was reaching the end of the section (`addr` or `bitmap` parsing was failing).